.changeset/polite-pillows-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-eslint/eslint-plugin': major
+---
+
+feat: `description-style` now have default description style `block`
```

Comment:
Was this an arbitrary change from `inline` to `block`, or is there motivation behind it?

Reply:
GitHub GraphQL public schema has `block` comments, so I changed this to `block`, also with `block` comments you can have multiline comments but with `inline` not

Reply:
Got it. Thus, requiring `inline` would be disallowing multiline comments.  Thanks for the details. 🙏